bettercli/cl/selector.py

```
@@ -146,31 +154,34 @@ def decorator(options:'list[str]'):
+            Cursor.write.hide_cursor()
+            self.keys = list(self.options.keys())
+            self.key = self.keys[0]
+
+            sys.stdout.write(ANSIColors.reset)
+            self.print()
+            keyboard.on_press(self.handler, suppress=True)
+            validated = False
+            while not validated:
+                while not self.ENTERED:
+                    pass
+                if (validate := self._validator(self.selected)) != True:
```

Ruff (0.8.2)

Avoid inequality comparisons to `True`; use `if not validate := self._validator(self.selected):` for false checks

Replace with `not validate := self._validator(self.selected)`

(E712)